From 7d35f6ea5f76c06a92ac4f6bb2b4da7c85c57241 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Tue, 26 Aug 2014 12:49:25 -0400
Subject: [tools] turbostat: remove unused command line option

Message-id: <1409057382-26432-15-git-send-email-prarit@redhat.com>
Patchwork-id: 89102
O-Subject: [RHEL7.1 PATCH BZ 1133893 14/31] tools/power turbostat: remove unused command line option
Bugzilla: 1133893
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1133893

commit f591c38b917fed99ac9cdfada84c3117ce4e0a1b
Author: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
Date:   Thu Jan 23 17:13:14 2014 +0200

    tools/power turbostat: remove unused command line option

    The -s is not used, let's remove it, and update quick help accordingly.

    Signed-off-by: Andy Shevchenko <andriy.shevchenko@linux.intel.com>
    Signed-off-by: Len Brown <len.brown@intel.com>

Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Dean Nelson <dnelson@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/power/x86/turbostat/turbostat.c b/tools/power/x86/turbostat/turbostat.c
index ed112ae..8c052cd 100644
--- a/tools/power/x86/turbostat/turbostat.c
+++ b/tools/power/x86/turbostat/turbostat.c
@@ -45,7 +45,7 @@ unsigned int verbose;  /* set with -v */
 unsigned int rapl_verbose; /* set with -R */
 unsigned int rapl_joules; /* set with -J */
 unsigned int thermal_verbose; /* set with -T */
-unsigned int summary_only; /* set with -s */
+unsigned int summary_only; /* set with -S */
 unsigned int skip_c0;
 unsigned int skip_c1;
 unsigned int do_nhm_cstates;
@@ -2098,7 +2098,7 @@ void check_cpuid()
 
 void usage()
 {
- errx(1, "%s: [-v][-R][-T][-p|-P|-S][-c MSR# | -s]][-C MSR#][-m MSR#][-M MSR#][-i interval_sec | command ...]\n",
+ errx(1, "%s: [-v][-R][-T][-p|-P|-S][-c MSR#][-C MSR#][-m MSR#][-M MSR#][-i interval_sec | command ...]\n",
       progname);
 }
 
@@ -2387,7 +2387,7 @@ void cmdline(int argc, char **argv)
 
  progname = argv[0];
 
- while ((opt = getopt(argc, argv, "+pPSvi:sc:sC:m:M:RJT:")) != -1) {
+ while ((opt = getopt(argc, argv, "+pPSvi:c:C:m:M:RJT:")) != -1) {
   switch (opt) {
   case 'p':
    show_core_only++;
-- 
1.7.1